src/app/[locale]/(landing)/page.tsx

```diff
@@ -372,14 +372,15 @@ export default async function LandingPage() {
 
 interface LanguageProgressStats {
   code: string;
-  name: string;
+  english_name: string;
+  local_name: string;
```

Comment:
You might also consider checking your queries to see if both names are needed and only load the one that's needed